src/appleseed-max-impl/appleseedinteractive/interactivetilecallback.cpp

```diff
@@ -32,6 +32,7 @@
 // 3ds Max headers.
 #include <bitmap.h>
 #include <interactiverender.h>
+#include <max.h>
```

Comment:
Why is this header required?

Reply:
max sdk 2017 complains about undefined Interface type. For some reason 2016 sdk compiles without any error.
I'll replace max.h with maxapi.h, it seems to bring less headers into the project.